src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -289,7 +289,7 @@ public function get_test_wordpress_version() {
 		} else {
 			foreach ( $core_updates as $core => $update ) {
 				if ( 'upgrade' === $update->response ) {
-					$current_version = explode( '.', $core_current_version );
+					$current_version = explode( '.', wp_get_wp_version() );
```

Comment:
@peterwilsoncc why change it here and not where `$core_current_version` is assigned?

```
$core_current_version = get_bloginfo( 'version' );
```

Reply:
No idea why I did it differently here, I've fixed this in f7416853952d7f1208f9d9cf6b80029fe81f519b